database/config.js

```diff
@@ -52,3 +52,20 @@ module.exports.getGame = (game, cb) => {
     }
   });
 };
+
+module.exports.postGame = (game, cb) => {
+  Game.insert(game, (err, result) => {
+    if (err) {
+      cb(err);
+    } else {
+      cb(null, result);
+    }
+  }
+};
+
+module.exports.deleteGame = (game, cb) => {
+
+};
+
+/////I'M CURIOUS ABOUT YA'LLS THOUGHTS ON THIS ^
+
```

Comment:
mongoose has a pretty simple delete function ... much like the insert one above.

Reply:
thank you Jay!